docs/how-to/using_rocm_bandwidth_test.rst

```diff
@@ -1,12 +1,14 @@
 .. meta::
-  :description: Using ROCm Bandwidth Test
-  :keywords: test, rocm bandwidth test, AMD, ROCm
+  :description: ROCm Bandwidth Test is a ROCm application for reporting system information
+  :keywords: ROCm bandwidth test usage, RBT usage, How to use RBT, How to use ROCm bandwidth test, ROCm bandwidth test user guide, RBT user guide, RBT user manual
 
 
 Using ROCm Bandwidth Test
 --------------------------
 
-The application ROCm Bandwidth Test (RBT) was developed to allow users discover the performance characteristics of Host-To-Device, Device-To-Host, and Device-to-Device copy operations on a ROCm platform. The application can be run on any compliant ROCm platform. The application provides various options for users to experiment the cost of various copy operations in both unidirectional and bidirectional modes. Users can query the various options that are supported by giving the `-h` option.
+RBT allows you to discover the performance characteristics of Host-To-Device, Device-To-Host, and Device-to-Device copy operations on a ROCm platform.
+RBT can be run on any ROCm-compliant platform and it provides various options to experiment with the various copy operation's costs in both unidirectional and bidirectional modes.
+You can query the various supported options using the `-h` option.
 
 Building ROCm Bandwidth Test
```

Comment:
I think this information overlaps with the information in the install.rst. Can we remove this section?  The build instructions given in install.rst are not exactly the same hence I am confused.

Reply:
Yes, we should be okay with that.

Reply:
Done.